static/report.js

```diff
@@ -146,7 +147,7 @@ function insertResults(results) {
 
 	let row = `
 	<div class="${res}">
-	    <span>${name}</span>
+	    <span><a href="${url}" target="_blank">${name}</a></span>
```

Comment:
Can you add `rel="noopener"` to the link (per [this](https://mathiasbynens.github.io/rel-noopener/))? The links are on trustworthy domains atm, but I want to make sure it's ok in the future.